pkg/volume/flexvolume/flexvolume.go

```diff
@@ -116,15 +117,15 @@ func (plugin *flexVolumePlugin) NewMounter(spec *volume.Spec, pod *api.Pod, _ vo
 			return nil, err
 		}
 		for name, data := range secretName.Data {
-			secret = string(data)
+			secrets[name] = base64.StdEncoding.EncodeToString(data)
```

Comment:
Just verifying, this method doesn't introduce newline characters in the encoded text does it?